astro/src/content/docs/apis/message-templates.mdx

```diff
@@ -138,7 +138,7 @@ This API does not return a JSON response body.
 
 This API is used to preview a Message Template. You pass all of the information for the Message Template in the request and a rendered version of the Message is sent back to you in the response.
 
-FusionAuth provides sample values for the `${code}` and `${user}` objects when previewing. The value for `${code}` will always be `123456`.
```

Comment:
I removed `${user}` because it's added to the preview but not used in any real message templates. I don't think we should make any code changes with this parameter.